study1/week5/group-anagrams.java

```diff
@@ -0,0 +1,41 @@
+// https://leetcode.com/problems/group-anagrams/
+// Time Complexity: O(nlogn), n: length of string which is the longest
+// Space Complexity: O(m+n), m: strMap.size, n: anagramList.size
```

Comment:
You are writing big O notation based on the output variables. You should rather write it based on the input variables, as the big O notation is reasoning about "how much is the complexity proportional to the input?"